GREEDY/ReceiptCreating/DataConverter.cs

```diff
@@ -76,10 +85,18 @@ public List<Item> ReceiptToItemList(Receipt receipt)
                     {
                         Name = m.Groups[1].Value.Replace("\n", string.Empty),
                         Price = decimal.Parse(m.Groups[2].Value.Replace(".", ",")),
-                        Category = ItemCategorization.CategorizeSingleItem(m.Groups[1].Value)
+                        Category = ""
                     });
                 }
             }
+            NewData = new List<ItemInfo>();
+            foreach (Item x in itemList)
+                NewData.Add(new ItemInfo("", x.Name, 0));
+            NewData = ItemCategorization.CategorizeAllItems(NewData);
+            foreach (Item x in itemList)
+                foreach (ItemInfo inf in NewData)
+                    if (inf.Text == x.Name)
+                        x.Category = inf.Category;
```

Comment:
need to create new method for categorization. also, our programming style needs brackets ({ }) for foreach

Reply:
Need to delete this part from there, create a new method and move to ItemCategorization class